plugins/exporter.koplugin/formats/html.lua

```diff
@@ -0,0 +1,64 @@
+local logger = require("logger")
+
+local slt2 = require('template/slt2')
+
+
+local HtmlExporter = require("formats/base"):new {
+    name = "html",
+}
+
+function HtmlExporter:prepareBookNotesForExport(booknotes)
```

Comment:
self contained functions like this one are better as local functions.

Reply:
Also, does this match the output of the previous exporter (before refactor)?

Reply:
Made it a local function.

Interestingly, in the previous plugin, there is code to show the chapter name, but it is not working. Here's the relevant lines:

https://github.com/koreader/koreader/blob/4fe39d6d4339ad74301f7e45282ad6d603b31232/plugins/exporter.koplugin/note.tpl#L62-L64

However, in the new plugin, it is working as expected. I'm attaching a zip file with two exports. The timestamped one is from the refactored plugin.

[Exporter examples.zip](https://github.com/koreader/koreader/files/8499976/Exporter.examples.zip)

Reply:
>  in the new plugin, it is working as expected.
Awesome. Thanks for the screenshots. 

Good change to keep

Reply:
Could you attach the before/after exports of the whole history?.

Reply:
Here's an example of the whole history. The newer export is timestamped, while exports from the older exporter are in the **old** directory.
[exporter-all-exports.zip](https://github.com/koreader/koreader/files/8503489/exporter-all-exports.zip)

Reply:
Thank you. All good :+1: